### PR TITLE
Api key generation

### LINKED ISSUE #48: Add new api keys/tokens
To the production database, for safety.

### PR DESCRIPTION
**Pull request by angaither**
Creates api key generation endpoint.
Must first be authenticated to create this (i.e. this will be done through Aurora)

@jonuy 

Fixes #48 
Refs DoSomething/mbc-userAPI-registration#10

### CODE REVIEW

File: app/controllers/KeyController.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  /**
+   * Store a newly created resource in storage.
+   * POST /keys
+   *
+   * @return Response
+   */
+  public function store()
+  {
+    // Get the app name from submission.
+    $app_name = Input::get('app_name');
```
Comment (jonuy) [inline app/controllers/KeyController.php]: Do you think app_name should be required? While testing I just created one where no app_name was provided.

**Comment by jonuy** [comment]
Just the one question above about maybe requiring app_name. Otherwise :+1:

**Comment by angaither** [comment]
@jonuy updated.... not sure how specific errors should be. meeeh

**Comment by jonuy** [comment]
I think that's sufficient :+1:

**Pull request merged**